Red November's 99th Anniversary

KrasnaMerah opened the pull request:
PKI Path
* Tan Malaka is added to the mod
* "PKI" focus would make Amir a communist-boosting advisor
* "Reorganize the PKI" would make Musso the leader of Underground PKI (he was here since December 1935) instead of going back to the Soviet Union
* Semaun's initial trait is changed to "Former Railroad Union Leader" and would return from the Soviet Union after the revolution is finished
* Jokosujono would be unlocked by "Reorganize the PKI" instead of "PKI" focus (he was in hiding in 1936)
* Added Party Congress events (3rd to 7th)

MISC Indonesia
* Hok Hoei Kan added back to the mod

MISC
* Philippines: Tan Malaka's trait is boosted and also has a non-LaR version (NCNS only)

Intended:
* More advisors
* Islamist and Fascist independence event chain
* More MIOs

KrasnaMerah committed:
Update 130725

Fixes for several misspelled character entry

Fixes for interface images

KrasnaMerah committed:
Update 130725 pt2

Werkplaats logo, further degenericize the Indies MIOs

marijn211 committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

BiccyThiccy committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

Commit:
Merge branch 'november-bugfixing-yay' into East-Indies-Reworked

KrasnaMerah committed:
Update 271025

* Tan Malaka is added to the mod
* "PKI" focus would make Amir a communist-boosting advisor
* "Reorganize the PKI" would make Musso the leader of Underground PKI instead of going back to the Soviet Union
* Semaun would only returning from the Soviet Union after the revolution is finished, his initial trait is "Former Railroad Union Leader"
* Jokosujono would be unlocked by "Reorganize the PKI" instead of "PKI" focus
* Added Party Congress event (3rd)

Commit:
fixing DEI bugs

*DEI = Dutch East Indies

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah commented:
Early review for now. Hopefully i can add more stuffs.

KrasnaMerah committed:
Update 111125

Sulawesi changes and more Party Congresses

marijn211 reviewed (changes requested):
I dont expect major conflicts but just to be sure lets hold this until after DLC patching

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

marijn211 commented on common/characters/INS.txt:
```diff
@@ -1184,6 +1205,55 @@ characters = {
+			idea_token = INS_tan_malaka
+			allowed = {
+				original_tag = INS
+				has_dlc = "Man the Guns"
```
Doesn't seem necessary? His trait doesn't affect anything MtG related

Tan Malaka has also been added on the Philippines in SEA, you could instance this character if you like the DLC portrait more

KrasnaMerah replied:
This is a leftover from EIR, as i'm locking several characters behind MtG.

Gotta delete this since MtG has been integrated

marijn211 commented on common/country_leader/r56_traits.txt:
```diff
@@ -3344,6 +3344,15 @@ leader_traits = {
+
+		ai_strategy = {
+			type = antagonize
+			id = "SOS"
```
SOS I believe only targets Stalinist USSR if they're in a civil war

KrasnaMerah replied:
Okay, i'm on it

KrasnaMerah committed:
Update 241125

Boosting Tan Malaka's trait for NCNS-PHI, fixing Congress and Indies CW events, readding Hok Hoei Kan

Commit:
Merge branch 'Beta-Edits' into East-Indies-Reworked

marijn211 commented on common/country_leader/00_traits.txt:
```diff
@@ -17242,6 +17242,17 @@ leader_traits = {
+		sprite = 13 
+
+		communism_drift = 0.05
+		boost_ideology_mission_factor = 0.15
```
I assume this is a LaR-exclusive modifier, though please confirm

Xaelorichero-02 replied:
It isn't. If you don't have it, it works on the old diplo actions to increase party popularity.
